src/features/currentTodo.ts

```diff
@@ -23,9 +23,16 @@ export const actions = { setTodo, removeTodo };
 type State = Todo | null;
 type Action = SetTodoAction | RemoveTodoAction;
 
+// eslint-disable-next-line @typescript-eslint/default-param-last
 const currentTodoReducer = (state: State = null, action: Action): State => {
   switch (action.type) {
-    // Implement all actions here
+    case 'currentTodo/SET': {
```

Comment:
for this case, you can also use a constant from one place with constants ( for all cases)